pom.xml

```diff
@@ -87,6 +88,11 @@
 		  <artifactId>lombok</artifactId>
 		  <version>1.16.10</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.5</version>
+		</dependency>
```

Comment:
Regular commons-lang in transient dependencies does not properly escape java. Hence the need for commons-lang3